packages/react-components/src/organisms/GroupTools.tsx

```diff
@@ -26,10 +27,13 @@ const button = css({
   },
 });
 
-type GroupToolsProps = Pick<GroupResponse, 'tools'>;
+type GroupToolsProps = Pick<GroupResponse, 'tools'> & {
+  readonly calendar?: GroupResponse['calendars'][0]['id'];
```

Comment:
`calendarId`, otherwise confusing given that other places call the whole object calendar